web/src/javascript/game.js

```diff
@@ -55,6 +55,12 @@ export function initMap(canvas, map) {
   canvasWidth = map.width;
   canvas.height = map.height;
   canvasHeight = map.height;
+  screenWidth = document.getElementById("gameDiv").offsetWidth - constants.scrollBarWidth;
+  screenHeight = screenWidth*0.6;
```

Comment:
Would be nice to have that `0.6` in a constant with a name, took me a moment to understand what it represents.